src/libcore/intrinsics.rs

```diff
@@ -224,15 +224,17 @@ extern "rust-intrinsic" {
     ///   trait objects, because they can't be read out onto the stack and
     ///   dropped normally.
     ///
-    /// * It is friendlier to the optimizer to do this over `ptr::read` when
+    /// * It is friendlier to the optimizer to do this over [`ptr::read`] when
     ///   dropping manually allocated memory (e.g. when writing Box/Rc/Vec),
```

Comment:
Can you add urls for Box, Rc and Vec as well please? (If you really don't want to, I won't bother you with it.)